src/components/ui/Paragraph.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from "react";
+import { VariantProps, cva } from "class-variance-authority";
+
+import { cn } from "@/lib/utils";
+
+export const paragraphVariants = cva(
+  "max-w-prose text-slate-700 dark:text-slate-300 mb-2 text-center",
+  {
+    variants: {
+      size: {
+        default: "text-2xl sm:text-3xl",        
+        lg: "text-3xl sm:text-4xl",
```

Comment:
@william-mao,
Please complete the responsive.

Reply:
okay got it.